cli/src/cli_util.rs

```diff
@@ -796,6 +796,7 @@ impl WorkspaceCommandHelper {
             return Ok(());
         }
 
+        writeln!(ui, "Imported changes from underlying Git repo")?;
```

Comment:
If we go with this, there should be a period after "repo". Also, perhaps this should be labeled so that we can make it brightblack or some other color?

Reply:
Another idea: perhaps put this after the messages about abandoned commits? The message could (optionally) be changed to emphasize the order of operations:

"Done importing changes from the underlying Git repo."

Reply:
My main motivation is to make it *really* clear that something like `jj describe` or `jj log` is not the reason commits got abandoned.

Reply:
Sounds good. Moved the message to the bottom.